### DISCUSSION TITLE
Search Option to allow searching of completed pos

### DISCUSSION THREAD

**Discussion by method1020** [inventree/InvenTree#4322] [Ideas]
I just realized that using the search tool does not show completed Purchase Orders. Am I missing it? It would be nice to allow searching of Completed items.

Thanks!

**Comment by SchrodingersGat** [comment]
@method1020 there is a configuration setting to control this:

![image](https://user-images.githubusercontent.com/10080325/217659925-61494e26-58ce-41a8-a96c-7d6eb06d6f9f.png)
